packages/design-system-scripts/gulp/docs/generatePages/generateDocPage.js

```diff
@@ -31,14 +31,14 @@ function generateDocPage(routes, page, docsPath, options) {
     return ReactDOMServer.renderToString(React.createElement(Docs, { page, routes: [] }, null));
   };
 
-  if (options.rootPath) options.rootPath = `${options.rootPath}/`;
+  const rootPath = (options.rootPath) ? `${options.rootPath}/` : null;
```

Comment:
If we make this an empty string instead of null, do we need all those `rootPath || ''` later on?